src/components/TasksTable/index.jsx

```diff
@@ -192,8 +247,14 @@ export default class TasksTable extends Component {
                 <TableCell component="th" scope="row">
                   {item.project}
                 </TableCell>
-                <TableCell>
-                  <List dense className={classes.summary}>
+                <TableCell className={classes.summary}>
+                  <Button
```

Comment:
I think we can use `IconButton` instead of `Button`. Also, add a margin right of 8px and make sure the "i" aligns with the summary text.